tests/python/relax/test_transform_fuse_ops.py

```diff
@@ -1001,7 +1001,7 @@ def main(
 
     @I.ir_module
     class Expected:
-        @T.prim_func
+        @T.prim_func(private=True)
```

Comment:
Throughout these Relax tests, I assumed that most PrimFuncs we will add through fusion, etc, are likely not intended to be public. Please let me know if this assumption is mistaken.